Makefile

```diff
@@ -16,6 +16,7 @@ BUILD_CONTAINER_NAME=calico/cni_build_container
 BUILD_CONTAINER_MARKER=cni_build_container.created
 DEPLOY_CONTAINER_NAME=calico/cni
 DEPLOY_CONTAINER_MARKER=cni_deploy_container.created
+TEMP_DIR:=$(shell mktemp -d)
```

Comment:
Would that all be done in the Dockerfile?  Would RUN be the best way to get the tar pulled and extraced?